ota-plus-web/docs/modules/ROOT/pages/create-fixed-group.adoc

```diff
@@ -0,0 +1,30 @@
+= Create a fixed group
+
+To create a fixed group, you need to specify the devices that you want to add. You can select the devices in the user interface or you can import a list of device IDs. If you want to include more than 10 devices, its probaby faster to import a list of device IDs.
```

Comment:
I don't think I agree with "If you want to include more than 10 devices, its probaby faster to import a list of device IDs" (also, n.b. typo on "probaby"), because of all the gotchas and caveats that you list below. Until we fix (at least) the following issues, importing device lists is a niche use case only:

* Feature doesn't work at all if user uploads a text file from Windows, an OS with about 80% of the global market
* Feature fails silently: user gets no notification of items that failed to import, meaning that to be able to rely on the feature to automate part of their workflow, they have to manually check every device in the group

That second one in particular means the feature is a foot gun. We shouldn't be writing documentation telling people to use foot guns at all, but if we do, we need to make sure it includes very clear warnings. Let's document the manual way of adding devices to groups, and have a separate "Advanced: Grouping devices by uploading a list" section that comes with strict warnings about how badly behaved the feature is.

Reply:
OK, I put the 'foot gun' doc in a collapsible section to keep it from distracting too much (e10ac58e8260134f4305018f5bb933529d16b4fd). Its true that this feature was badly thought out (implemented for one specific customer I think)